app/models/game.rb

```diff
@@ -0,0 +1,13 @@
+class Game
+  include Mongoid::Document
+
+  field :winner, type: String
```

Comment:
you should validate that this is in the expected list. e.g it's one of 'team_one', 'team_two', and 'tie' or whatever